README.md

```diff
@@ -114,13 +152,12 @@ osd:
         aws_access_key_id: your_aws_access_key_id
         aws_secret_access_key: your_aws_secret_access_key
       coprhd:
-        restUrl: coprhd_rest_url
-        user: rest_user_name
-        password: rest_user_password
-        consistency_group: consistency_group_id
-        project: project_id
-        varray: varray_id
-        vpool: vpool_id
+        url: corphd_rest_url
+        token: coprhd_proxy_token
+        port: host_initiator_port_name
+	project: opt_default_project_name
```

Comment:
Im looking at the rendered README, I don't see anything wrong with the indentation. Everything looks aligned properly.